app/models/comment.rb

```diff
@@ -90,6 +90,10 @@ def is_deliverable?
   def truncate_user_agent
     self.user_agent = user_agent.try(:slice, 0, 255)
   end
+
+  def body_snippet
+    body.truncate(100)
```

Comment:
It's probably a bit nicer to use Rails' truncate stuff as a view helper: https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-truncate

Reply:
My gut says since the truncate helper is there, let's use it. I don't intuitively understand why we'd want to add code/abstraction to accomplish the same end result, but take that with a grain of salt and feel free to explain! I'm sorta a vanilla-rails person who prioritizes claritty and straightforwardness and gets confused easily with patterns and so on. My day job is basically unravelling patterns people add to their rails apps to regain performance/clarity :D I do think there are a few of places in alonetone in need of encapsulation (for example user names / profile pictures)...